content/fr/docs/setup/release/building-from-source.md

```diff
@@ -29,6 +29,6 @@ cd kubernetes
 make release
 ```
 
-Pour plus de détails sur le processus de release, voir le repertoire [`build`](http://releases.k8s.io/{{< param "githubbranch" >}}/build/) dans kubernetes/kubernetes.
+Pour plus de détails sur le processus de release, voir le repertoire [`build`](http://releases.k8s.io/master/build/) dans kubernetes/kubernetes.
```

Comment:
This file is not existed in English version.

Reply:
/hold

Reply:
> It's inconsistent with English version. /hold

Yes it is not consistent, but at least it points to an actual link.
This PR is just to fix the dead links, and do not intend to re-update the doc content.
The later could be done in another PR.